### PR TITLE
feat: add statements get command and print out statement details and …

### PR DESCRIPTION
**Pull request by swist**
Release Notes
-------------
New Features
- adds a command for viewing statement details and any exceptions that it might have thrown

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----
add statements get command and print out statement details and its exceptions if present

References
----------

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

File: internal/cmd/flink/command_statement_describe.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+func (c *command) newStatementGetDetailsCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe <name>",
+		Short:             "Get details of a Flink SQL statement.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validStatementArgs),
+		RunE:              c.statementDescribe,
+	}
+
+	pcmd.AddForceFlag(cmd)
```
Comment (YannickPferr) [inline internal/cmd/flink/command_statement_describe.go:18]: Isn't that only used for delete statements?

File: internal/cmd/flink/command_statement_exceptions_list.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+func (c *command) newStatementExceptionsListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "list <statement name>",
+		Short:             "Get exceptions for a Flink SQL statement.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validStatementArgs),
+		RunE:              c.exceptionsList,
+	}
+
+	pcmd.AddForceFlag(cmd)
```
Comment (YannickPferr) [inline internal/cmd/flink/command_statement_exceptions_list.go:18]: Isn't that only used for delete statements?

**Review by YannickPferr — APPROVED** [review]
One question about the forceFlag and I think you need to fix the lint errors, otherwise LGTM!

File: internal/cmd/flink/command_statement_exceptions_list.go (hunk 2 above)
Comment (swist) [reply internal/cmd/flink/command_statement_exceptions_list.go:18]: good catch, removed

**Pull request closed without merging**